tests/std/tests/P0355R7_calendars_and_time_zones_clocks/test.cpp

```diff
@@ -378,6 +378,19 @@ static_assert(is_clock_v<tai_clock>);
 static_assert(is_clock_v<gps_clock>);
 static_assert(is_clock_v<file_clock>);
 
+tzdb copy_tzdb() {
+    const auto& my_tzdb = get_tzdb_list().front();
+    vector<time_zone> zones;
+    vector<time_zone_link> links;
+    _STD transform(my_tzdb.zones.begin(), my_tzdb.zones.end(), _STD back_inserter(zones),
+        [](const auto& _Tz) { return time_zone{_Tz.name()}; });
+    _STD transform(my_tzdb.links.begin(), my_tzdb.links.end(), _STD back_inserter(links), [](const auto& link) {
+        return time_zone_link{link.name(), link.target()};
+    });
+
+    return {my_tzdb.version, _STD move(zones), _STD move(links), my_tzdb.leap_seconds, my_tzdb._All_ls_positive};

```

Comment:
```suggestion
    transform(my_tzdb.zones.begin(), my_tzdb.zones.end(), back_inserter(zones),
        [](const auto& _Tz) { return time_zone{_Tz.name()}; });
    transform(my_tzdb.links.begin(), my_tzdb.links.end(), back_inserter(links), [](const auto& link) {
        return time_zone_link{link.name(), link.target()};
    });

    return {my_tzdb.version, move(zones), move(links), my_tzdb.leap_seconds, my_tzdb._All_ls_positive};
```
Since we're `using namespace std;` in this file, we should be able to drop the `_STD` here!

Reply:
copy and paste artifact, will update.